package.json

```diff
@@ -45,11 +51,32 @@
     "yeoman-generator": "^1.1.1"
   },
   "eslintConfig": {
-    "extends": "xo-space",
+    "extends": [
+      "xo-space",
```

Comment:
xo-space isn't needed here.. already interfered by `eslint-config-prettier` and `eslint-plugin-prettier`

Reply:
`eslint-config-prettier` only turn off style related rules.

`xo-space` add other linting rules. That being said, we probably could just use the `xo` preconfigured rules.

Reply:
Yeah, but we're also using `eslint-plugin-prettier` which prefers space by default. It makes `xo-space` useless to be used here, doesn't it?

Reply:
`xo-space` is the `xo` config + space preference. So yeah, we don't specifically need xo-space, we can use `xo` directly.

Reply:
ah my bad, I misread this..
you're using `xo` as a `eslint` shareable config, not directly 😞

Reply:
but yeah, why use eslint-config-xo-space instead of eslint-config-xo? directly?